### ISSUE TITLE
Locales no longer working

### ISSUE THREAD

**Issue by andrewdotn** [js-joda/js-joda#366]
Copying-and-pasting [the first example in the docs](https://github.com/js-joda/js-joda/tree/master/packages/locale#es5):

```
const {
    DateTimeFormatter,
    ZonedDateTime,
    ZoneId,
} = require('@js-joda/core');
require('@js-joda/timezone');

const {
    Locale,
} = require('@js-joda/locale_en-us');

var zdt = ZonedDateTime.of(2016, 1, 1, 0, 0, 0, 0, ZoneId.of('Europe/Berlin'));
console.log('en_US formatted string:', zdt.format(DateTimeFormatter.ofPattern('eeee MMMM dd yyyy GGGG, hh:mm:ss a zzzz, \'Week \' ww, \'Quarter \' QQQ').withLocale(Locale.US)));
```

with minimal dependencies in `package.json`:

```
{
  "dependencies": {
    "@js-joda/core": "^1.12.0",
    "@js-joda/locale_en-us": "^2.0.1",
    "@js-joda/timezone": "^2.2.0"
  }
}
```

    $ node example.js
    node_modules/@js-joda/core/dist/js-joda.js:5920
            throw new IllegalArgumentException('Pattern using (localized) text not implemented, use js-joda-locale plugin!');
            ^
    IllegalArgumentException: Pattern using (localized) text not implemented, use js-joda-locale plugin!
        at DateTimeFormatterBuilder.appendText (node_modules/@js-joda/core/dist/js-joda.js:5920:15)
        at DateTimeFormatterBuilder._parseField (node_modules/@js-joda/core/dist/js-joda.js:6186:22)
        at DateTimeFormatterBuilder._parsePattern (node_modules/@js-joda/core/dist/js-joda.js:5997:20)
        at DateTimeFormatterBuilder.appendPattern (node_modules/@js-joda/core/dist/js-joda.js:5910:14)
        at Function.ofPattern (node_modules/@js-joda/core/dist/js-joda.js:6656:47)
        at Object.<anonymous> (example.js:13:69)
        at Module._compile (internal/modules/cjs/loader.js:956:30)
        at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
        at Module.load (internal/modules/cjs/loader.js:812:32)
        at Function.Module._load (internal/modules/cjs/loader.js:724:14) {
      message: 'Pattern using (localized) text not implemented, use js-joda-locale plugin!',
      toString: [Function]
    }

it blows up saying to use the deprecated package name.

This worked before the `js-joda` → `@js-joda/core` transition.

Please advise on how to format a date in en_US locale using the `@joda` packages.

**Comment by pithu** [comment]
Try this one, works for me
``` 
npm install @js-joda/locale_en-us@3.0.0
```
for some reasons `npm install @js-joda/locale_en-us` installs a wrong, older version.

**Comment by phueper** [comment]
It seems the @js-joda/locale_en-us@3.0.0 is published with the beta tag, see : https://www.npmjs.com/package/@js-joda/locale_en-us  ...  in the "Version" tab..

@andrewdotn could you please verify that it works with the explouct @js-joda/locale_en-us@3.0.0 install... then we can verify to make it latest instead of beta 

Regards, Pattrick

**Comment by andrewdotn** [comment]
Yup, that works for me! Thanks for figuring this out so quickly.

**Comment by phueper** [comment]
Hi,

we just released @js-joda/locale 3.0.1 tagged as "latest" ... the workaround with the explicit version number should no longer be necessary.

I'll close this issue

Regards, Pattrick
